### PR TITLE
feat: unify trade directions

### PR DESCRIPTION
**Pull request by koonopek**

### CODE REVIEW

**Comment by cehali** [comment]
@koonopek can we do this after: https://github.com/redstone-finance/redstone-oracles-monorepo/pull/874, I am resolving conflicts right now and other changes to balancer fetchers will make it really hard for me

File: packages/utils/src/types/redstone-metadata.ts [hunk 1]
```diff
@@ -20,6 +20,12 @@ export interface MetadataPerSource {
 
 export interface SlippageData {
   slippageAsPercent: string;
-  direction: "buy" | "sell";
+  direction: TradeDirections;
   simulationValueInUsd: string;
 }
+
+export type TradeDirections = "buy" | "sell";
```
Comment (rpiszczatowski) [inline packages/utils/src/types/redstone-metadata.ts:27]: Consider
```
export type TradeDirections = `${TradeDirection}`
```
not to repeat yourself.

**Review by rpiszczatowski — APPROVED** [review]
Nice cleanup

**Review by cehali — APPROVED** [review]
Good refactor. You have minor conflicts to resolve, `BalancerMultiFetcher` is now `BalancerFetcher`

File: packages/utils/src/types/redstone-metadata.ts (hunk 1 above)
Comment (koonopek) [reply packages/utils/src/types/redstone-metadata.ts:27]: nice one, didnt know about it

**Review by cehali — APPROVED** [review]
LGTM

**Review by rpiszczatowski — APPROVED** [review]

**Pull request merged**